src/main/java/com/sugarcrm/candybean/automation/webdriver/WebDriverElement.java

```diff
@@ -358,9 +359,27 @@ public void scroll(Location loc) throws CandybeanException {
 	 * element, so links will be activated, radio buttons will be selected, drop boxes will be
 	 * opened, etc.  Use of this method on any non-textfield element is not currently supported.
 	 */
-	public void selectAll() {
-		we.click();
-		we.sendKeys(Keys.END,Keys.chord(Keys.SHIFT,Keys.HOME));
+	public void selectAll() throws CandybeanException {
+		executeJavascript("arguments[0].select()");

```

Comment:
Thank you @jmittert  for answering the question. This is due to the fact that this `executeJavascript` is in `WebDriverElement`, so the `executeJavascript` in this method already has a web element as the first argument.